RUFAS/input_manager.py

```diff
@@ -91,3 +91,78 @@ def _load_data(self) -> None:
                                    info_map)
             except Exception as e:
                 raise e
+
+    def _validate_data(self, eager_termination: bool = True) -> bool:
+        """
+        Validates input data.
+
+        Parameters
+        ----------
+        eager_termination : bool, default=True
+            Flag to determine if the process should terminate upon finding invalid data.
```

Comment:
Specify the expected behavior.
```suggestion
            If true, the process will be terminated upon finding invalid data.
```